### PR TITLE
treewide: don't reference aliases in packages

### PR DESCRIPTION
**Pull request by samuelgrf**
###### Motivation for this change
Fix building with `config.allowAliases = false;`.

The only package rebuilds this causes are because of removing references to `perlPackages.TestMore` in https://github.com/NixOS/nixpkgs/commit/e5deebdf53c25cc38a58fb0bf8d51f0548503a04. This does not affect the packages in any way because TestMore has been merged into Perl and the alias already defines it as `null`:

https://github.com/NixOS/nixpkgs/blob/496963f42819f4e20f5eea08da4c04b054aa4ce0/pkgs/top-level/perl-packages.nix#L23851
https://github.com/NixOS/nixpkgs/blob/496963f42819f4e20f5eea08da4c04b054aa4ce0/pkgs/top-level/perl-packages.nix#L23783

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by samuelgrf** [comment]
Result of `nixpkgs-review pr 116564` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>convos</li>
    <li>perl530Packages.CatalystAuthenticationStoreLDAP</li>
    <li>perl530Packages.NetLDAPServerTest</li>
    <li>perl532Packages.CatalystAuthenticationStoreLDAP</li>
    <li>perl532Packages.NetLDAPServerTest</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 116564` at a5ea64ea run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built successfully:</summary>
  <ul>
    <li><a href="https://gist.github.com/bc48b45f72df54f275e578985ca23588">convos</a></li></li>
    <li><a href="https://gist.github.com/bc0382ef00b7aafd201f32936bb2c196">perl530Packages.CatalystAuthenticationStoreLDAP</a></li></li>
    <li><a href="https://gist.github.com/a30e109fefc1dcf2d3dc0e47c9407eb1">perl530Packages.NetLDAPServerTest</a></li></li>
    <li><a href="https://gist.github.com/b590f51a20a77fdc8b03628cac7ab2f7">perl532Packages.CatalystAuthenticationStoreLDAP</a></li></li>
    <li><a href="https://gist.github.com/854d66c2ffa7d3510f591cea40c1ab3a">perl532Packages.NetLDAPServerTest</a></li></li>
  </ul>
</details>

**Review by jonringer — APPROVED** [review]
diff LGTM

looks lilke rebuilds are mostly from removing an unneeded dependency

File: pkgs/applications/networking/cluster/nixops/shell.nix [hunk 1]
```diff
@@ -3,7 +3,7 @@
 pkgs.mkShell {
   buildInputs = [
     pkgs.poetry2nix.cli
-    pkgs.pkgconfig
+    pkgs.pkg-config
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/cluster/nixops/shell.nix:6]: Does this need to be changed in the nixops repo?

File: pkgs/development/node-packages/default.nix [hunk 2]
```diff
@@ -107,7 +107,7 @@ let
     mirakurun = super.mirakurun.override rec {
       nativeBuildInputs = with pkgs; [ makeWrapper ];
       postInstall = let
-        runtimeDeps = [ nodejs ] ++ (with pkgs; [ bash which v4l_utils ]);
+        runtimeDeps = [ nodejs ] ++ (with pkgs; [ bash which v4l-utils ]);
```
Comment (SuperSandro2000) [inline pkgs/development/node-packages/default.nix:110]: That will get overwritten with the next update.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 116564` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
5 packages built:
</summary>
<ul>
<li>convos</li>
<li>perl530Packages.CatalystAuthenticationStoreLDAP</li>
<li>perl530Packages.NetLDAPServerTest</li>
<li>perl532Packages.CatalystAuthenticationStoreLDAP</li>
<li>perl532Packages.NetLDAPServerTest</li>
</ul>
</details>

**Pull request merged**